### ISSUE TITLE
Thumbnails are not displaying correctly in checklist

### ISSUE THREAD

**Issue by arbolitoloco** [arbolitoloco/NEON-Biorepository#5]
https://biorepo.neonscience.org/portal/checklists/checklist.php?clid=45&amp;pid=1&amp;dynclid=0

https://neonbiorepository.slack.com/archives/CJ6DTEKJP/p1557338080008400

Reported by Ed Gilbert.

**Comment by arbolitoloco** [comment]
it's actually the printoff div's size that is changing and pushing things down.
this should fix it:

```
#cloptiondiv {
   margin: 5px;
   min-width: 250px;
   text-align: left;
   float: right;
   height: 316px;
}
```

**Comment by arbolitoloco** [comment]
And fixed in 03634b26 and f3c24e5b
